app/models/manager_refresh/inventory/automation_manager.rb

```diff
@@ -0,0 +1,33 @@
+module ManagerRefresh::Inventory::AutomationManager
+  extend ActiveSupport::Concern
+  include Core
+
+  class Inventory < ManagerRefresh::Inventory::Core::Inventory
+    def credentials
+      inventory authentication,
+                :model_class    => ManageIQ::Providers::AutomationManager::Authentication,
+                :association    => :credentials,
+                :builder_params => {
+                  :resource => ->(persister) { persister.manager }
```

Comment:
@Ladas this is another way to access the `persister` at instance level. We could do [this](https://github.com/ManageIQ/manageiq/pull/14022/files#diff-1588bc23945c2827748c43d7c14a3b0cR24) for all options passed